test/services/LibraryService.test.ts

```diff
@@ -317,6 +317,34 @@ describe('LibraryService', () => {
         });
     });
 
+    it('throws a 400 error when an id is included in both the path and a fhir parameter', async () => {
```

Comment:
```suggestion
    it('throws a 400 error when an id is included in both the path and a FHIR parameter', async () => {
```

Reply:
Maybe add something to the end about it being for the $package operation. Same for the other description below

Reply:
This is in the package operation test set, so you should end up with something like:
`LibraryService › $package › throws a 400 error when an id is included in both the path and a FHIR parameter`
(similar with below)